sunpy/map/header_helper.py

```diff
@@ -52,6 +56,14 @@ def make_fitswcs_header(data, coordinate, reference_pixel: u.pix = None,
         Pixel scaling along x and y axis (i.e. the spatial scale of the pixels (dx, dy)). These are
         expected to be Cartestian ordered, i.e [dx, dy].
         Defaults to ``([1., 1.] arcsec/pixel)``.
+    rotation_angle : `~astropy.unit.Quantity`, optional
+        Coordinate system reference angle i.e. the rotation of the pixel array wrt the WCS.
+    rotation_matrix_pc : `~np.ndarray` of dimensions 2x2, optional
+        Matrix describing the rotation required to align solar North with
+        the top of the image in PCi_j convention.
+    rotation_matrix_cd : `~np.ndarray` of dimensions 2x2, optional
```

Comment:
I suggest we don't support this. Map can't support having `CDi_j` without `scale` which means you have the scale infomation duplicated in the matrix and the `CDELT` parameters. So I would just support adding `PCi_j`.

Reply:
ok will fix